app/export_test.go

```diff
@@ -377,11 +377,32 @@ func TestExportDMandGMPost(t *testing.T) {
 	gmMembers := []string{th1.BasicUser.Username, user1.Username, user2.Username}
 
 	// DM posts
-	th1.CreatePost(dmChannel)
-	th1.CreatePost(dmChannel)
+	th1.App.CreatePost(&model.Post{
```

Comment:
nit: Propose pulling down `model.Post` onto its own line, and indent its members one more than the rest of the function. Lastly, pull down the closing `)` onto its own line too.

Reply:
Done. Thanks for pointing that out.
I got this from other test and I took it from granted as it being somehow a blessed pattern.